assets/js/event-providers/popup-maker.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * Site Kit by Google, Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+document.addEventListener( 'DOMContentLoaded', () => {
```

Comment:
As in [this comment](https://github.com/google/site-kit-wp/pull/8651/files/c7dbbeccd9244f566c059da7695b7f5e2cd50659#r1591482308) we can't assume that DOMContentLoaded hasn't already happened since the script is loaded async. Can you remove this event listener in favour of `( ( jQuery ) => {` and set the script to `defer` rather than `async` below.